h/static/scripts/admin-users.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+function AdminUsersController(element) {
+  this._form = element.querySelector('#js-users-delete-form');
+
+  function confirmFormSubmit() {
+    return window.confirm("This will permanently delete all the user's data. Are you sure?");
+  }
+
+  if(this._form !== null) {
```

Comment:
In JS code we just use truthiness tests rather than testing for exactly null vs. undefined etc., so just `if (this._form)`.